libcomposefs/lcfs-internal.h

```diff
@@ -29,6 +29,28 @@
  */
 #define LCFS_BUILD_INLINE_FILE_SIZE_LIMIT 64
 
+#define OVERLAY_XATTR_USER_PREFIX "user."
```

Comment:
Aweseome, I think this helps a lot to have these.  Then we can also followup with some links to the kernel sources.